homeassistant/components/tuya/sensor.py

```diff
@@ -739,6 +739,74 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             icon="mdi:progress-clock",
         ),
     ),
+    # Air Purifier
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48r41mn81
+    "kj": (
+        TuyaSensorEntityDescription(
+            key=DPCode.FILTER,
+            name="Filter Utilization",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            icon="mdi:ticket-percent-outline",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.FILTER_DAYS,
+            name="Remaining days of the filter",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            icon="mdi:ticket-confirmation-outline",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.PM25,
+            name="Particulate Matter 2.5 µm",
+            device_class=SensorDeviceClass.PM25,
+            state_class=SensorStateClass.MEASUREMENT,
+            icon="mdi:molecule",
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TEMP,
+            name="Temperature",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.HUMIDITY,
+            name="Humidity",
+            device_class=SensorDeviceClass.HUMIDITY,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TVOC,
+            name="Total Volatile Organic Compound",
+            device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.ECO2,
+            name="Concentration of Carbon Dioxide",
+            device_class=SensorDeviceClass.CO2,
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TOTAL_TIME,
+            name="Total Operating Time",
+            icon="mdi:history",
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.TOTAL_PM,
+            name="Total Absorption of Particles",
+            icon="mdi:texture-box",
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.AIR_QUALITY,
+            name="Air quality",
+            icon="mdi:air-filter",
+            device_class=TuyaDeviceClass.AIR_QUALITY,
+            state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
This is not a numeric value for this device?

Reply:
In the documentation an Enum with the quality assessment is defined.

<html>
<body>
<!--StartFragment-->

air_quality | Indoor air quality | Enum | {“range”:[“great”,“mild”,“good”,“medium”,“severe”]}
-- | -- | -- | --


<!--EndFragment-->
</body>
</html>

I'm going to remove the StateClass that was left there incorrectly.